### PR TITLE
Animated URL template tag

### PR DESCRIPTION
**Pull request by MichaelButkovic**
@mparent61 @benghaziboy

### CODE REVIEW

File: djbetty/templatetags/betty.py [hunk 1]
```diff
@@ -23,6 +23,9 @@ def __init__(self, id):
     def get_crop_url(self, ratio="original", width=600, format="jpg", fixed=0):
         return self.storage.url(self.id, ratio=ratio, width=width, format=format, fixed=fixed)
 
+    def get_animated_url(self, format="gif", fixed=0):
```
Comment (mparent61) [inline djbetty/templatetags/betty.py]: Might as well change default `fixed=0` in all places to `fixed=False` (for both animated + cropped methods). Previous author was using `0` for a boolean flag, which is confusing.

File: djbetty/storage.py [hunk 2]
```diff
@@ -99,3 +99,23 @@ def url(self, name, ratio="original", width=600, format="jpg", fixed=0):
...
+            if index % 4 == 0 and index != 0:
+                id_string += "/"
+            id_string += char
+
+        # Allows request to use fixed url from settings
+        base_url = getattr(settings, 'BETTY_FIXED_URL', None) if fixed else None
+        if not base_url:
+            base_url = self.base_url
+
+        base_url = base_url.rstrip('/')
```
Comment (mparent61) [inline djbetty/storage.py]: First part of this function shares identical code with `def url`, I would move into a common helper function (esp tricky logic to build id_string)

**Comment by MichaelButkovic** [comment]
@mparent61 @benghaziboy ok, did a slight refactor based on comments

**Comment by benghaziboy** [comment]
👍

**Comment by mparent61** [comment]
Awesome 👍

**Pull request merged**